portal/src/graphql/portal/LockoutSettings.tsx

```diff
@@ -20,7 +20,7 @@ export interface State {
   historyDurationMins?: number;
   minimumDurationMins?: number;
   maximumDurationMins?: number;
-  backoffFactor?: number;
+  backoffFactorRaw?: string;
```

Comment:
Why do we need to use string to represent this?

Reply:
@louischan-oursky If we allow the user to type decimal places, the user can type something like "3.", which if we parse it into number it becomes `3`, and if we put it back into the TextField it becomes "3". As a result the user can never type ".". So, we should store the current input value as string.

Reply:
> In fact I always think form text field value states should always be string, even it represents a number.

Agree.